clients/web/src/utilities/EventStream.js

```diff
@@ -64,6 +73,7 @@ prototype.open = function () {
                 stream.trigger('g:error', e);
                 return;
             }
+            window.localStorage.setItem('sseTimestamp', obj._girderTime);
```

Comment:
For robustness, I'd consider catching exceptions from `setItem()`. From https://developer.mozilla.org/en-US/docs/Web/API/Storage/setItem:

> setItem() may throw an exception if the storage is full. Particularly, in Mobile Safari (since iOS 5) it always throws when the user enters private mode (Safari sets quota to 0 bytes in private mode, contrary to other browsers, who allow storage in private mode, using separate data containers).
> Hence the developers should make sure to **always catch the possible exceptions from setItem()**.
>